src/contributing/void-docs/style-guide.md

```diff
@@ -6,6 +6,19 @@ This style guide outlines the standards for contributing to the
 [mdBook](https://rust-lang-nursery.github.io/mdBook/) stored in the
 [void-docs](https://github.com/void-linux/void-docs/) repository.
 
+## General
+
+Step-by-step instructions containing 'magic' commands for copying-and-pasting
+are strongly discouraged. Users are expected to read the canonical documentation
```

Comment:
This could probably be qualified with "outside of the installation guides section" just to be honest. We need to be careful there still but in the end, we should still provide a set of commands that can be combined to get Void installed.

Reply:
Good point. Changed to: "Outside of the 'installation' sections, step-by-step instructions ..."